em_workflows/utils/flow.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+"""
+Utilities workflows:
+-------------------
+These workflows are created simply to aid developers.
+
+- Use #1: Run the tests regularly.
+  Test input files can be of big size which can be hard to manage. Since most of the tests are normally executed on
+  HPC over local machine, it is deemed okay have a prefect scheduler to run the tests for us.
+"""
+import subprocess
+
+from prefect import flow, task
+from prefect.artifacts import create_markdown_artifact
+
+from em_workflows.config import Config
+from em_workflows.utils import utils
+
+
+@task
+def publish_artifact(report: str) -> None:
+    """
+    The report will be published in the prefect server under Artifacts section
+    """
+    create_markdown_artifact(
+        key="pytest-cov-report",
+        markdown=report,
+        description="Pytest Coverage Report",
+    )
+
+
+@task
+def run_tests() -> str:
+    """
+    Run pytest and save coverage
```

Comment:
maybe include:
returned `str` is visible at https://prefect2.hedwig-workflow-api.niaiddev.net/artifacts/key/pytest-cov-report